src/views/virtualmachines/tree/hooks/dragndrop.ts

```diff
@@ -51,46 +51,49 @@ export const addDragEventListener = (treeViewItem: TreeViewDataItem): RemoveList
 };
 
 export const dropEventListeners = (treeViewItem: TreeViewDataItem): RemoveListenerFunction => {
-  const folderHTMLElement = document.getElementById(treeViewItem.id);
+  const dropHTMLElement = document.getElementById(treeViewItem.id);
 
-  const [_, folderNamespace, folderName] = treeViewItem.id.split('/');
+  const [_, dropNamespace, folderName] = treeViewItem.id.split('/');
 
-  if (!folderHTMLElement || !folderName) return null;
+  if (!dropHTMLElement) return null;
 
-  const dropHandler = (ev) => {
-    ev.preventDefault();
+  const dropHandler = (event) => {
+    event.stopPropagation();
+    event.preventDefault();
     // Get the id of the target and add the moved element to the target's DOM
-    const vmID = ev.dataTransfer.getData('application/treeViewElementID');
+    const vmID = event.dataTransfer.getData('application/treeViewElementID');
 
     const [namespace, name] = vmID.split('/');
 
     changeVMFolder(name, namespace, folderName);
 
-    folderHTMLElement.style.backgroundColor = 'unset';
+    dropHTMLElement.style.backgroundColor = 'unset';
```

Comment:
Would you mind moving these strings ('unset', 'move', 'none') to constants or an enum?